specification/paths.json

```diff
@@ -113,6 +113,9 @@
   "/regions/{region_id}": {
     "$ref": "./paths/Regions-region_id.json"
   },
+  "/register-shipment": {
```

Comment:
I'm not convinced about this endpoint name. I believe it may cause confusion to users, making them think they need to call this endpoint in order to register any shipment with the carrier.
Maybe `create-and-register-shipment`?

Reply:
I created a hidden `POST /registered-shipments` endpoint with `application/vnd.api+json` as discussed.